tutorials/00_getting_started_flexible.ipynb

```diff
@@ -63,7 +64,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For this illustrative example, we consider a model _simulator_, that takes in 3 parameters ($\\theta$). For simplicity, the _simulator_ outputs simulations of the same dimensionality and just adds 1.0 and some Gaussian noise to the parameter set. \n",
+    "For this illustrative example, we consider a model _simulator_, that takes in 3 parameters ($\\theta$). For simplicity, the _simulator_ outputs simulations of the same dimensionality and adds 1.0 and some Gaussian noise to the parameter set. \n",
     "\n",
     "> Note: This is where you would specify your model _simulator_ and with its parameters. \n",
```

Comment:
This was not changed, but the sentence needs to be corrected i.e. "This is where you instead would use your specific _simulator_ with its parameters."

Reply:
changed it